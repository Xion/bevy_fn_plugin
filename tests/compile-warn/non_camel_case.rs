#![deny(non_camel_case_types)]

#[macro_use] extern crate bevy_fn_plugin;

use bevy::prelude::*;

#[bevy_plugin]
fn non_camel_case(_: &mut App) {}

fn main() {}
