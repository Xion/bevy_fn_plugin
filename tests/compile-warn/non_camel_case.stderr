warning: unused import: `bevy::prelude::*`
 --> tests/compile-warn/non_camel_case.rs:5:5
  |
5 | use bevy::prelude::*;
  |     ^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error: type `non_camel_case` should have an upper camel case name
 --> tests/compile-warn/non_camel_case.rs:8:4
  |
8 | fn non_camel_case(_: &mut App) {}
  |    ^^^^^^^^^^^^^^ help: convert the identifier to upper camel case: `NonCamelCase`
  |
note: the lint level is defined here
 --> tests/compile-warn/non_camel_case.rs:1:9
  |
1 | #![deny(non_camel_case_types)]
  |         ^^^^^^^^^^^^^^^^^^^^
